common/avrdude.conf

```diff
@@ -11670,6 +11670,197 @@ part
       ;
   ;
 
+#------------------------------------------------------------
+# ATmega16u4
```

Comment:
For some reason this *still* isn't in avrdude upstream...

Reply:
Tested in visual studio with live atmega32a and confirmed working.